recipe/meta.yaml

```diff
@@ -1,24 +1,24 @@
-{% set version = "4.6.0" %}
+{% set version = "4.7.0" %}
 
 package:
   name: pexpect
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pexpect/pexpect-{{ version }}.tar.gz
-  sha256: 2a8e88259839571d1251d278476f3eec5db26deb73a70be5ed5dc5435e418aba
+  sha256: 9e2c1fd0e6ee3a49b28f95d4b33bc389c89b20af6a1255906e90ff1262ce62eb
 
 build:
-  number: 1000
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
   build:
     - python
     - pip
   run:
     - python
-    - ptyprocess  # [not win]
+    - ptyprocess >=0.5  # [not win]
```

Comment:
@takluyver should we skip windows entirely instead of just the `ptyprocess` dep?

Reply:
Pexpect kind of supports Windows, although the most widely used part doesn't:

https://pexpect.readthedocs.io/en/stable/overview.html#pexpect-on-windows